oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -71,6 +72,9 @@
 public class ComputeEngineCredentials extends GoogleCredentials
     implements ServiceAccountSigner, IdTokenProvider {
 
```

Comment:
Could you add some comments on why we had to change the margins specifically for GCE?

Reply:
done